test/integration/change-streams/change_stream.test.ts

```diff
@@ -702,15 +703,19 @@ describe('Change Streams', function () {
 
       const outStream = new PassThrough({ objectMode: true });
 
-      // @ts-expect-error: transform requires a Document return type
-      changeStream.stream({ transform: JSON.stringify }).pipe(outStream);
+      const transform = doc => ({ doc: JSON.stringify(doc) });
+      changeStream
```

Comment:
unhandled rejections fix (and just change the transform to remove the TS error)

Reply:
Why was this unhandled not caught in https://github.com/mongodb/node-mongodb-native/pull/4448?

Reply:
These didn't reject previously because we weren't closing checked-out connections